### PR TITLE
issue#15 create not found page

### PR DESCRIPTION
**Pull request by arya56**
# Description
Creating Not found page. 
Please provide a short summary explaining what this PR is about.
1.It's been said to add stroybook but I couldn't find any logic to create a storybook for any element inside this page. 
2.Related route has been set for this page ( in the app.js line:35)
#sceenshot
image

Fixes # (issue)
#issue 15
# How to test?
run the project and try to put a non-existed route in the browser address bar under the project local host.
Please provide a short summary how your changes can be tested?
check this out : http://localhost:3000/Ali_eh
# Checklist

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

**Comment by arya56** [comment]
Not found page_issue#15

File: .storybook/main.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+module.exports = {
```
Comment (pdsorensen) [inline .storybook/main.js:1]: @arya56 can you remove this file? Unless there is a reason for it to be here. :)

File: .storybook/preview.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
```
Comment (pdsorensen) [inline .storybook/preview.js:1]: @arya56 can you remove this file? Unless there is a reason for it to be here. :)

Comment (arya56) [reply .storybook/preview.js]: Done

File: .storybook/main.js (hunk 1 above)
Comment (arya56) [reply .storybook/main.js]: Done!

**Review by pdsorensen — APPROVED** [review]
Looks good to me. @benna100 can i have you confirm if this is how we add fonts, i.e in the `src/client/index.css` file.

**Pull request merged**